model/src/main/java/io/gravitee/definition/model/Proxy.java

```diff
@@ -15,28 +15,37 @@
  */
 package io.gravitee.definition.model;
 
+import com.fasterxml.jackson.annotation.*;
+
 import java.io.Serializable;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
+import java.util.stream.Collectors;
 
 /**
  * @author David BRASSELY (david.brassely at graviteesource.com)
  * @author GraviteeSource Team
  */
+@JsonIgnoreProperties({ "endpoint", "multiTenant" }) // TODO this is currently tested?!
```

Comment:
@phiz71 
Why are there fields tested, which are not part of the model?
Should I clean up the corresponding tests?